### PR TITLE
Change: Infantry lasered by the USA Paladin Tank will burn

### PR DESCRIPTION
**Pull request by xezon**
* Relates to #1848
* Relates to #2067

With this change all infantry lasered by the USA Paladin Tank will burn. This change is cosmetic only.

## Patched

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/4720891/3d51ba70-6c95-4ee1-9367-fec5ad5058b4

### CODE REVIEW

**Comment by commy2** [comment]
Video shows the bad behaviour that will be fixed with #2072

File: Patch104pZH/GameFilesEdited/Data/INI/Object/CivilianUnit.ini [hunk 1]
```diff
@@ -442,7 +442,7 @@ Object MogadishuMaleCivilian01
   End
 
   Behavior = FXListDie ModuleTag_07
-    DeathTypes = ALL -CRUSHED -SPLATTED -POISONED_BETA -POISONED -BURNED -POISONED_GAMMA
+    DeathTypes = ALL -CRUSHED -SPLATTED -POISONED_BETA -POISONED -BURNED -LASERED -POISONED_GAMMA
     DeathFX = FX_CivilianArabMaleDie
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/CivilianUnit.ini:446]: Just a comment: Strange that these use FXListDie::DeathFX when the other files all used SlowDeathBehavior::FX instead.

#2074 fixes that

**Review by commy2 — APPROVED** [review]
my god the pain

**Comment by xezon** [comment]
No pain no gain.

**Comment by xezon** [comment]
Documentation added. Looks good.

**Pull request merged**